app/src/main/res/layout/activity_main.xml

```diff
@@ -26,9 +26,10 @@
         android:id="@+id/start_recommendation"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="bottom|right"
+        android:layout_gravity="bottom|end"
         android:layout_margin="@dimen/fab_margin"
         android:tint="@color/white"
-        android:src="@android:drawable/ic_input_add"/>
+        android:src="@android:drawable/ic_input_add"
+        app:layout_behavior="com.belatrixsf.allstars.ui.common.ScrollAwareFABBehavior"/>
 
```

Comment:
have you tried this behavior with what develop have? Last thing i remember I was having some issues between the tab bars and Floating action button in the coordinator